src/settings/Passwd.js

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import { Form, Field } from 'react-final-form';
+
+import { stripesConnect } from '@folio/stripes/core';
+import {
+  Button,
+  Callout,
+  Headline,
+  Pane,
+  TextField
+} from '@folio/stripes/components';
+
+/**
+ * Reset a user's password
+ */
+class Passwd extends React.Component {
+  static manifest = Object.freeze({
+    passwd: {
+      type: 'okapi',
+      path: 'authn/credentials',
+      fetch: false,
+      accumulate: true,
+    },
+    userExists: {
+      type: 'okapi',
+      path: 'users',
+      fetch: false,
+      accumulate: true
+    },
+    isLocalPasswordSet: {
+      type: 'okapi',
+      path: 'authn/credentials-existence',
+      fetch: false,
+      accumulate: true,
+    },
+    moduleId: '',
+  });
+
+  static propTypes = {
+    stripes: PropTypes.shape({
+      connect: PropTypes.func.isRequired,
+    }).isRequired,
+    resources: PropTypes.shape({
+      passwd: PropTypes.object,
+      isLocalPasswordSet: PropTypes.object,
+    }).isRequired,
+    mutator: PropTypes.shape({
+      passwd: PropTypes.object,
```

Comment:
C'mon, if this is going to have any value, it has to be `shape({ ... methods ... })`.

(Or, if we increasingly think it _doesn't_ have value, let's be honest about it and just stop using prop-types, instead of shoving `object` in to shut it up.)

Reply:
Also, you need to declare `mutator.userExists.GET` here, too, don't you?